proposed/phpdoc-tags.md

```diff
@@ -644,13 +644,9 @@ RECOMMENDED in case of complicated return structures, such as associative arrays
 The @return tag MAY have a multi-line description and does not need explicit
 delimiting.
 
-It is RECOMMENDED to use this tag with every function and method. An exception to
-this recommendation, as defined by the Coding Standard of any individual project,
-MAY be:
-
-   **functions and methods without a `return` value**: the @return tag MAY be
-   omitted here, in which case an interpreter MUST interpret this as if
-   `@return void` is provided.
+It is RECOMMENDED to use this tag with every function and method. The @return
```

Comment:
I would suggest to keep the recommendation in the specification to minimum. For example here lot of people may argue that it's pointless to add `@return int` to methods with native `int` return type.

Reply:
Doesn't this recommendation also contradict the "This document SHALL NOT ... Describe best practices or recommendations for Coding Standards on the application of the PHPDoc standard."?